packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -111,17 +115,43 @@ class ${class_name}:
         self._pin_map_client = pin_map_client
         self._stub: Optional[v2_measurement_service_pb2_grpc.MeasurementServiceStub] = None
         self._measure_response: Optional[
-            Generator[v2_measurement_service_pb2.MeasureResponse, None, None]
+            grpc.CallIterator[v2_measurement_service_pb2.MeasureResponse]
```

Comment:
This type is defined by https://github.com/shabbyrobe/grpc-stubs/ , not by `grpcio`:

```
>>> import grpc
>>> grpc.CallIterator
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'grpc' has no attribute 'CallIterator'
```

Somehow, this particular annotation is not causing a run time error, but the same annotation in a function signature would cause a run time error:

```
>>> def f() -> grpc.CallIterator: pass
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'grpc' has no attribute 'CallIterator'
```

Please add `from __future__ import annotations` to the generated code, just to be safe.

Reply:
I have added `from __future__ import annotations` import to the generated code.